community/php7-ssh2/APKBUILD

```diff
@@ -3,31 +3,52 @@
 pkgname=php7-ssh2
 _pkgreal=ssh2
 pkgver=1.1.2
-pkgrel=0
+pkgrel=1
 pkgdesc="PHP extension provide bindings for the libssh2 library"
 url="http://pecl.php.net/package/ssh2"
 arch="all"
 license="PHP"
 depends=""
 pecldepends="php7-dev autoconf"
 makedepends="$pecldepends libssh2-dev"
+options="!check" # todo: Generate keypair and setup of user
```

Comment:
the only question for me... why no check and `check()` added bellow?

Reply:
!check - because the test will only work when the user and cert files have been generated.

I added the check() function to document what that have to be done. I did not have the time at the time to complete it.

Reply:
@TBK are you gonna finish it?

Reply:
I have been trying to solve it but since it requires either root priviledges or an already created user in the CI enviroment I do not know how to complete it:

```
check() {
        cd "$builddir"

        # Setup user and cert
        adduser -D -g "" sshuser
        echo "sshuser:sshpassword" | chpasswd
        mkdir -p ~sshuser/.ssh
        chown sshuser:sshuser ~sshuser/.ssh
        sh -c "cat $(pwd)/tests/testkey_rsa.pub >> ~sshuser/.ssh/authorized_keys"
        chown sshuser:sshuser ~sshuser/.ssh/authorized_keys
        chmod 600 ~sshuser/.ssh/authorized_keys

        # Run dropbear
        dropbear

        export "TEST_SSH2_HOSTNAME=localhost"
        export "TEST_SSH2_USER=sshuser"
        export "TEST_SSH2_PASS=sshpassword"

        make NO_INTERACTION=1 REPORT_EXIT_STATUS=1 test

        # Kill dropbear
        pkill dropbear

        # Cleanup
        rm -rf ~sshuser
        deluser sshuser
}
```

Without the dropbear part, 6/7 tests will succeed.